src/titiler/core/titiler/core/templates/index.html

```diff
@@ -69,79 +36,63 @@
     }
 }
 
-const addAOI = (bounds) => {
-  if (map.getLayer('aoi-polygon')) map.removeLayer('aoi-polygon')
-  if (map.getSource('aoi')) map.removeSource('aoi')
-  const geojson = {
-      "type": "FeatureCollection",
-      "features": [bboxPolygon(bounds)]
+// tms comes from morecantile
+// https://github.com/developmentseed/morecantile/blob/main/morecantile/models.py
+var crs = new L.Proj.CRS(
+    '{{ tms.crs.srs }}',
+    '{{ tms.crs.to_proj4() }}', {
+    origin: [{{ tms.xy_bbox[0] }}, {{ tms.xy_bbox[3] }}],
+    // bounds: [[{{ tms.xy_bbox[0] }}, {{ tms.xy_bbox[1] }}], [{{ tms.xy_bbox[2] }}, {{ tms.xy_bbox[3] }}]],
+    resolutions: {{ resolutions|safe }},
   }
+);
 
-  map.addSource('aoi', {
-    'type': 'geojson',
-    'data': geojson
-  })
 
-  map.addLayer({
-    id: 'aoi-polygon',
-    type: 'line',
-    source: 'aoi',
-    layout: {
-      'line-cap': 'round',
-      'line-join': 'round'
-    },
-    paint: {
-      'line-color': '#3bb2d0',
-      'line-width': 1
-    }
-  })
-  return
-}
+var map = L.map('map', {
```

Comment:
While you're debugging, add a few control points to make sure the CRS mess is working properly. I tend to use stuff like

```
const madrid = L.marker([40, -3]).addTo(map);
```

Also, something very useful is:

```
map.on('click', (ev)=>console.log(ev.latlng));
```